src/test/scala/uk/gov/nationalarchives/consignmentexport/GraphQlApiSpec.scala

```diff
@@ -32,7 +32,12 @@ class GraphQlApiSpec extends ExportSpec {
     val updateExportClient = mock[GraphQLClient[uel.Data, uel.Variables]]
     val keycloak = mock[KeycloakUtils]
     val api = new GraphQlApi(keycloak, consignmentClient, updateExportClient)
-    val config = Configuration(S3("", "", ""), Api(""), Auth("authUrl", "clientId", "clientSecret", "realm"), EFS(""))
+    val config = Configuration(
+      S3("", "", ""),
+      Api(""),
+      Auth("authUrl", "clientId", "clientSecret", "realm"),
+      EFS(""),
+      SFN(""))
```

Comment:
Is it worth extracting this out the top of the class? It seems to be much the same for each of the tests?

Reply:
Fair point